server/athenian/api/precompute/accounts.py

```diff
@@ -487,8 +501,56 @@ def track(self, account: int, meta_ids: Sequence[int], is_fresh: bool) -> None:
         metrics.precompute_account_seconds.clear()
         metrics.precompute_account_seconds.labels(
             account=account,
-            github_account=",".join(map(str, sorted(meta_ids))),
+            github_account=_github_account_tracking_label(meta_ids),
             is_fresh=is_fresh,
         ).observe(elapsed)
         self._log.info("tracking precompute duration: %.3f seconds", elapsed)
-        push_metrics(self._gateway, self._PROMETHEUS_JOB)
+        push_metrics(self._gateway)
+
+
+class _ResolutionTracker:
```

Comment:
```suggestion
class _StatusTracker:
```

I recommend renaming "resolution" to "status" because it's a more common term in this context. "Resolution" is commonly used with timers, e.g. Python's default timer's resolution is 1 microsecond. Another meaning is: this problem's or issue's resolution is "Won't fix".

GHA defines "status" and "conclusion" of a check run, where "status" is `SUCCESS`, `FAILURE`, etc.

Reply:
done